src/form/Select/SelectMenu/SelectMenu.tsx

```diff
@@ -44,6 +44,14 @@ export interface SelectMenuProps {
    */
   createable?: boolean;
 
+  /**
+   * Function to render the create option.
+   */
+  renderCreateOption?: (
```

Comment:
Let's make this a object so its more flexible in case we need to extend it in the future.